src/main/resources/category_templates.json

```diff
@@ -163,4 +163,23 @@
   "_id" : "INSCROA",
   "categoryName" : "Change a company’s registered office address",
   "parent" : "INS"
-} ]
+}, {
+  "_id" : "INSCROA",
+  "categoryName" : "Change a company’s registered office address",
+  "parent" : "INS"
+}, {
+  "_id" : "INSCROA",
+  "categoryName" : "Change a company’s registered office address",
+  "parent" : "INS"
+}, {
```

Comment:
did you mean to duplicate these INSCROA categories?

Reply:
Good spot!